components/MobileMenu/index.tsx

```diff
@@ -21,8 +21,8 @@ const MobileMenu = ({ links }: MobileMenu) => {
             leaveFrom='transform scale-100 opacity-100 translate-y-0'
             leaveTo='transform scale-95 opacity-0 translate-y-10'
           >
-            <Popover.Panel className='flex flex-col items-end'>
-              <div className='bg-primary flex justify-around flex-col items-center p-3 rounded-lg w-full h-96 font-serif'>
+            <Popover.Panel className='flex flex-col items-end justify-end h-screen'>
+              <div className='bg-primary flex justify-around flex-col items-center p-3 rounded-lg w-full h-[50%] font-serif'>
```

Comment:
there's already a class that does half height
```suggestion
              <div className='bg-primary flex justify-around flex-col items-center p-3 rounded-lg w-full h-1/2 font-serif'>
```